### PR TITLE
Better approach to transfer call with inline NCCO

### PR DESCRIPTION
**Pull request by superdiana**
Modified the transfer call with inline ncco with new approach that shows how this works in real time

### CODE REVIEW

File: voice/transfer-call-inline-ncco.py [hunk 1]
```diff
@@ -1,40 +1,42 @@
...
+envpath = join(dirname(__file__),'./.env')
 load_dotenv(envpath)
 
-# Init the client
+#Init the client
 print(os.getenv("NEXMO_PRIVATE_KEY"))
 client = nexmo.Client(
-    application_id=os.getenv('NEXMO_APPLICATION_ID'),
-    private_key=os.getenv("NEXMO_PRIVATE_KEY")
+    application_id = os.getenv('NEXMO_APPLICATION_ID'),
```
Comment (lornajane) [inline voice/transfer-call-inline-ncco.py]: I think this should be in the `example.env` file to prompt people to set this value

File: voice/transfer-call-inline-ncco.py [hunk 2]
```diff
@@ -1,40 +1,42 @@
...
 response = client.create_call({
-    "to": [{"type": "phone", "number": os.getenv('TO_NUMBER')}],
-    "from": {"type": "phone", "number": os.getenv('FROM_NUMBER')},
-    "ncco": [
-        {
-            "action": "talk",
-            "text": "This is just a text whilst you tranfer to another NCCO"
-        }
-    ]
+  'to': [{'type': 'phone', 'number': os.getenv('TO_NUMBER' }],
```
Comment (lornajane) [inline voice/transfer-call-inline-ncco.py]: `os.getenv()` is missing its closing parenthesis here (also the one on the next line!)

File: voice/transfer-call-inline-ncco.py (hunk 1 above)
Comment (lornajane) [reply voice/transfer-call-inline-ncco.py]: (also the `NEXMO_PRIVATE_KEY`)

Comment (superdiana) [reply voice/transfer-call-inline-ncco.py]: Corrected. And Adding to example.env

**Comment by superdiana** [comment]
@lornajane  the rule is that as long as the call is in course, the speech should be replaced

**Comment by superdiana** [comment]
Resolved

File: voice/transfer-call-inline-ncco.py [hunk 3]
```diff
@@ -8,30 +8,21 @@
...
-    "to": [{"type": "phone", "number": os.getenv('TO_NUMBER')}],
-    "from": {"type": "phone", "number": os.getenv('FROM_NUMBER')},
-    "ncco": [
-        {
-            "action": "talk",
-            "text": "This is just a text whilst you tranfer to another NCCO"
-        }
-    ]
-})
+uuid = os.getenv('CALL_UUID')
```
Comment (dragonmantank) [inline voice/transfer-call-inline-ncco.py:17]: This should be `UUID`, per the [specs](https://github.com/Nexmo/code-snippet-specs/blob/master/voice/transfer-a-call-with-ncco.md#variables)

File: voice/transfer-call-inline-ncco.py [hunk 4]
```diff
@@ -8,30 +8,21 @@
...
-    "to": [{"type": "phone", "number": os.getenv('TO_NUMBER')}],
-    "from": {"type": "phone", "number": os.getenv('FROM_NUMBER')},
-    "ncco": [
-        {
-            "action": "talk",
-            "text": "This is just a text whilst you tranfer to another NCCO"
-        }
-    ]
-})
+UUID = os.getenv('CALL_UUID')
```
Comment (slorello89) [inline voice/transfer-call-inline-ncco.py:17]: env var should match the var name be called `UUID`

File: example.env [hunk 5]
```diff
@@ -10,6 +10,10 @@ INSIGHT_NUMBER='NUMBER_FOR_INSIGHT'
 # Account
 SMS_CALLBACK_URL=
 
+# Voice
+NEXMO_APPLICATION_ID='YOUR_APPLICATION_ID"
+NEXMO_PRIVATE_KEY='PATH_TO_KEY_FILE"
+
```
Comment (slorello89) [inline example.env:16]: add `UUID` to example.env

File: voice/transfer-call-inline-ncco.py (hunk 4 above)
Comment (superdiana) [reply voice/transfer-call-inline-ncco.py:17]: This is a standard for 5 code snippets I'm just following after what is already done.

**Pull request merged**